src/spikeinterface/postprocessing/tests/common_extension_tests.py

```diff
@@ -133,6 +133,11 @@ def _check_one(self, sorting_analyzer, extension_class, params):
         sliced = sorting_analyzer.select_units(some_unit_ids, format="memory")
         assert np.array_equal(sliced.unit_ids, sorting_analyzer.unit_ids[::2])
 
+        some_merges = [sorting_analyzer.unit_ids[:2].tolist()]
+        num_units_after_merge = len(sorting_analyzer.unit_ids) - 1
+        merged = sorting_analyzer.merge_units(some_merges, format="memory", mode="soft", sparsity_overlap=0.0)
+        assert len(merged.unit_ids) == num_units_after_merge
```

Comment:
This is really a small test no ?

Reply:
Yes, we could/should extend that

Reply:
big test will be implemented in postprocessing/test_multiextension